### PR TITLE
Setup sound triggers

### PR DESCRIPTION
**Pull request by Jokubas126**
Added music clips and configured the AudioTrigger for it to be able to interupt music together with narrative

### CODE REVIEW

File: Assets/Scripts/EndTaskInteraction.cs [hunk 1]
```diff
@@ -74,7 +76,8 @@ void FixedUpdate()
             {
                 playerMovementRecorder.endTaskCompleted = true;
                 taskCompleted = true;
-                audioPlayer.PlayAudioClip(endTaskClip, false);
+                audioPlayer.PlayAudioClip(endTaskClip, false, null);
```
Comment (latedude2) [inline Assets/Scripts/EndTaskInteraction.cs]: Why you pass null here? You should be able to just not have the third argument

**Review by latedude2 — CHANGES_REQUESTED** [review]

File: Assets/Scripts/StartTest.cs [hunk 2]
```diff
@@ -21,15 +23,17 @@ void Start()
         subtitles = GetComponentInChildren<Subtitles>();
         audioPlayer = GetComponent<AudioScript>();
 
+        audioPlayer.PlayAudioClip(startMusicClip, true, 0.4f);
+
         if (isArrowCondition)
         {
             subtitles.SetupUpSubtitles(subtitleTextArrow, subtitleTimeArrow);
-            audioPlayer.PlayAudioClip(introClipArrow, false);
+            audioPlayer.PlayAudioClip(introClipArrow, false, null);
```
Comment (latedude2) [inline Assets/Scripts/StartTest.cs]: Same here

File: Assets/Scripts/StartTest.cs [hunk 3]
```diff
@@ -21,15 +23,17 @@ void Start()
...
         } 
         else
         {
             subtitles.SetupUpSubtitles(subtitleTextBee, subtitleTimeBee);
-            audioPlayer.PlayAudioClip(introClipBee, false);
+            audioPlayer.PlayAudioClip(introClipBee, false, null);
```
Comment (latedude2) [inline Assets/Scripts/StartTest.cs]: same here

File: Assets/Scripts/EndTaskInteraction.cs (hunk 1 above)
Comment (Jokubas126) [reply Assets/Scripts/EndTaskInteraction.cs]: Yeah, I thought of having two separate functions for the same thing with different attributes, but that was also an option in my mind 🤔 do you think the approach with two functions would be better?

File: Assets/Scripts/StartTest.cs (hunk 2 above)
Comment (Jokubas126) [reply Assets/Scripts/StartTest.cs]: explained above ☝️

File: Assets/Scripts/StartTest.cs (hunk 3 above)
Comment (Jokubas126) [reply Assets/Scripts/StartTest.cs]: explained above ☝️

File: Assets/Scripts/EndTaskInteraction.cs (hunk 1 above)
Comment (Jokubas126) [reply Assets/Scripts/EndTaskInteraction.cs]: Because music clips have to get some volume adjustment and narration shouldn't

Comment (Jokubas126) [reply Assets/Scripts/EndTaskInteraction.cs]: So maybe just passing `1f` would be better? :D

**Review by latedude2 — APPROVED** [review]

**Pull request merged**